demo/NostrSDKDemo/Views/ConnectRelayView.swift

```diff
@@ -11,20 +11,28 @@ import Combine
 
 struct ConnectRelayView: View {
 
+    @Binding var relay: Relay?
```

Comment:
Why do we need a `Binding` to a relay rather than just a relay?

Reply:
I am passing it in from the `ContentView` so each view that needs it can share the relay. But, maybe that's not necessary for it to be a `@Binding`. I basically just default to that when I pass a `@State` to another object.